lib/nexpose/vuln_exception.rb

```diff
@@ -3,31 +3,69 @@ module Nexpose
   class Connection
     include XMLUtils
 
-    # Retrieve vulnerability exceptions.
+    # Retrieve all active vulnerability exceptions.
```

Comment:
I suggest expanding the docs to explain the limitations of this method, and that not all values returned by it are supported in other methods that still use the XML API.